.rubocop_todo.yml

```diff
@@ -14,7 +14,7 @@ Gemspec/OrderedDependencies:
   Exclude:
     - 'core/solidus_core.gemspec'
 
-# Offense count: 197
+# Offense count: 201
```

Comment:
why did this went up?

Reply:
Because I don't personally care about the method alignment cop, and every time I move the arguments to a `has_many` call into multiple lines, it starts complaining. StandardRB doesn't care either. :)

Reply:
Ah! Reminds me of https://github.com/solidusio/solidus/pull/6241 😆